src/query/mapquerydirective.js

```diff
@@ -119,4 +123,11 @@ ngeo.mapQueryDirective = function(ngeoMapQuerent, $injector) {
   };
 };
 
-ngeo.module.directive('ngeoMapQuery', ngeo.mapQueryDirective);
+
+ngeo.query.mapQueryDirective.module = angular.module('ngeoMapQuery', [
+  ngeo.query.MapQuerent.module.name,
+]);
+
+ngeo.module.requires.push(ngeo.query.mapQueryDirective.module.name);
+
+ngeo.query.mapQueryDirective.module.directive('ngeoMapQuery', ngeo.query.mapQueryDirective);
```

Comment:
I think we said to export the Angular module for directives and components (not the directive function).
```
ngeo.query.mapQueryDirective = angular.module('ngeoMapQuery', [
  ngeo.query.MapQuerent.name,
]);
```

Reply:
+1
This file is strange. That should be:
`ngeo.query.mapQueryDirective = angular.module...`
`ngeo.module.requires....`
...
`ngeo.query.mapQueryDirective.directive_ = function...`
...
`ngeo.query.mapQueryDirective.directive('ngeoMapQuery', ngeo.query.mapQueryDirective.directive_);`

no ?